sullivan_a07b.js

```diff
@@ -0,0 +1,90 @@
+/*  Anna Sullivan
+    sullivan_a07b.js
+    INFO2124WW
+    Thoendel
+    2/1/2020
+*/
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+const standardInput = process.stdin;            
+standardInput.resume();
+standardInput.setEncoding('utf8');
+const badFoods = [];
+const groceryItems = [];
+clearScreen();
+console.log(getPrompt(groceryItems));
+/* DO NOT MODIFY CODE BETWEEN THESE LINES */
+
+//Step 1
+buildFoodsList (badFoods, "bread", "beer", "cheese", "milk", "cookie");     /*added 3 more string values as arguments*/   
+
+standardInput.on('data', function (text) {
+    //This line will remove line breaks 
+    //\n\r in window, \r in macOS
+    text = text.replace('\n', '').replace('\r', '');
+    //Step 4 Begin
+
+    /*if else to determine the user input using decision stucture*/
+if((text == "q") || (text == "Q")){
+    clearScreen();
+    console.log(`Bye!`);
+    process.exit();
+}else{
+    clearScreen();
+    checkItem(text, badFoods);
+    if(checkItem() == false){
+        text += formatItem();

```

Comment:
this is going to append the formatted version - though you aren't passing anything to format. Try:
text = formatItem(text);

Reply:
then you'll also need to actually add the item to the groceryItems[] (try .push() or .unshift())